1657408: Sixth set of 20 new SEO descriptions. (#18848)
dotnet/docs: docs/standard/asynchronous-programming-patterns/implementing-the-event-based-asynchronous-pattern.md

---
title: "Implementing the Event-based Asynchronous Pattern"

> description: In this article, learn to implement the Event-based Asynchronous Pattern (EAP) in .NET. EAP is a standard way to package a class having asynchronous features.

Reviewer: ```suggestion
description: Learn how to implement the Event-based Asynchronous Pattern (EAP) in .NET. EAP is a standard way to package a class that has asynchronous features.
```
